packages/honeycomb-opentelemetry-web/src/validate-options.ts

```diff
@@ -33,32 +33,48 @@ export const FAILED_AUTH_FOR_LOCAL_VISUALIZATIONS =
   );
 
 export const validateOptionsWarnings = (options?: HoneycombOptions) => {
+  const logLevel = options?.logLevel
```

Comment:
Q for this change: Are there any other areas we'd like to see the logs being controlled?

Reply:
If we search for `console.*` Looks like there's some statements in `packages/honeycomb-opentelemetry-web/src/console-trace-link-exporter.ts`. 
We might need to tweak the API for `ConsoleTraceLinkExporter`  to accept the full `options` object rather than just the service name.